src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/WsTrust/WsTrustResponse.cs

```diff
@@ -148,9 +148,17 @@ internal static WsTrustResponse CreateFromResponseDocument(XDocument responseDoc
                     {
                         continue;
                     }
-                    
-                    tokenResponseDictionary.Add(tokenTypeElement.Value,
-                        requestedSecurityToken.FirstNode.ToString(SaveOptions.DisableFormatting));
+
```

Comment:
Are there no tests for this method? Could you write some - it shouldn't be too difficult since it's a static method that takes in just an xml and a version. Boy scout rule :)

Reply:
Test case added. Am I a good boy now? :)